crates/types/src/config/worker.rs

```diff
@@ -310,6 +310,18 @@ pub struct InvokerOptions {
     #[cfg_attr(feature = "schemars", schemars(skip))]
     #[serde(skip_serializing_if = "std::ops::Not::not", default)]
     experimental_features_allow_protocol_v6: bool,
+    /// # Action throttling
+    ///
+    /// Configures rate limiting for service actions at the node level.
+    /// This throttling mechanism uses a token bucket algorithm to control the rate
+    /// at which actions can be processed, helping to prevent resource exhaustion
+    /// and maintain system stability under high load.
+    ///
+    /// The throttling limit is shared across all partitions running on this node,
+    /// providing a global rate limit for the entire node rather than per-partition limits.
+    /// When set to `None`, no throttling is applied and actions are processed
```

Comment:
Note that `None` is not a valid value in toml. So it's better to describe it in terms of being set or unset.